src/setup/cortx_deploy

```diff
@@ -94,13 +101,51 @@ validate_config $CLUSTER_INFO
 
 export PATH=$PATH:/opt/seagate/provisioner/bin
 
-# Apply Cluster Config
-log $INFO "cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c '$CORTX_CONFSTORE'"
-cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c "$CORTX_CONFSTORE";
+# Check system is pre-configured:
+# config apply: check confstore file.
+# cluster bootstrap: check VERSION file is exist and phase and status key from confstore and.
+
+node_id=`cat /etc/machine-id`
+cluster_id=`conf $(yaml_config $CLUSTER_INFO) get "cluster>id"`
+cluster_id=$(echo $cluster_id | tr -d "["\" | tr -d "\"]")
+log $INFO "Starting deployment on node $node_id which is part of cluster $cluster_id."
 
-# Apply Common Config
-log $INFO "cortx_setup config apply -f $(yaml_config $CONFIG_INFO) -c '$CORTX_CONFSTORE'"
-cortx_setup config apply -f $(yaml_config $CONFIG_INFO) -c "$CORTX_CONFSTORE";
+FILE=$(awk -F:// '{print $2}' <<< $CORTX_CONFSTORE)
+if [ -f "$FILE" ]; then
+    log $WARNING "$FILE file exists. System looks to be pre-configured."
+    CONFIG_APPLY=false
+
+    version_file=`conf $CORTX_CONFSTORE get "node>$node_id>version"`
+    version_file=$(echo $version_file | tr -d "["\" | tr -d "\"]")
```

Comment:
This logic should be simple 

version=$(conf_get "node>node_id>version" 2> /dev/null)
[ ! -z $version ] && [ -z $FORCE_DEPLOY ] && echo "error: CORTX is already configured on this node" && exit 1

Proceed with normal installation. No need to refer to $FORCE_DEPLOY again

Reply:
Removed this code from cortx_deploy.